apps/hash-frontend/src/blocks/page/block-context-menu/block-context-menu.tsx

```diff
@@ -47,6 +62,11 @@ const BlockContextMenu: ForwardRefRenderFunction<
   { blockEntity, deleteBlock, openConfigMenu, popupState, canSwap },
   ref,
 ) => {
+  const { blockSubgraph, setBlockSubgraph } = useBlockContext();
+  const fetchBlockSubgraph = useFetchBlockSubgraph();
+
+  const [open, setOpen] = useState(false);
```

Comment:
This context menu has many items in it – I'd call this `entityEditorIsOpen` or something

Reply:
Changed here https://github.com/hashintel/hash/pull/1809/commits/916515b842b88016061425376e0fa7d75c5290fb